### PR TITLE
book updates

### PR DESCRIPTION
**Pull request by arnetheduck**
* systemd: use same service file as package
* systemd: recommend setting up a user
* clean up obsolete / ephemeral notes
* eth1 -> execution, in many places
* clean up docker instructions to match the experience of running the
runner script
* add basic instructions / links for execution clients other than geth
* clear a few more references to JSON-RPC, update to REST
* update hardware requirements with execution client extras
* start talking about merge requirements (this needs a dedicated page)
* note that binaries do exist, where relevant
* list gitcoin under donations
* update syncing docs
* `NOT` is for `NOTICE` in logs
* update rpi hardware recommendations
* prevent automatic doppelganger restarts in systemd service file

### CODE REVIEW

File: docs/the_nimbus_book/src/docker.md [hunk 1]
```diff
@@ -6,19 +6,23 @@ We have version-specific Docker tags (`statusim/nimbus-eth2:amd64-v1.2.3`) and a
...
+Before running nimbus via docker, you need to prepare a data directory and mount it in docker.
+
+It is recommended that you mount the directory at `/home/user/nimbus-eth2/build/data` and pass `--data-dir=build/data/shared_mainnet_0` to all `nimbus_becaon_node` commands.
+
+The wrapper script outlined below will set the data directory automatically.
 
 ```text
 mkdir data
-docker run -it --rm -v ${PWD}/data:/home/user/nimbus-eth2/build/data statusim/nimbus-eth2:amd64-latest [nimbus_beacon_node args here]
+docker run -it --rm -v ${PWD}/data://home/user/nimbus-eth2/build/data statusim/nimbus-eth2:amd64-latest --data-dir=build/data/shared_mainnet_0 --network=mainnet [other options]
```
Comment (zah) [inline docs/the_nimbus_book/src/docker.md]: You added an extra slash here in front of the path here (`//home`). Was it intentional?

File: docs/the_nimbus_book/src/faq.md [hunk 2]
```diff
@@ -34,26 +45,26 @@ You can think of it as a small penalty that you pay only on first launch and res
...
-We recommend [running a Nimbus beacon node](./quick-start.md) on [Prater](./prater.md) and a mainnet [eth1 client](./eth1.md) on the same machine.
+We recommend running [a Nimbus beacon node](./quick-start.md) on [Prater](./prater.md) and a mainnet [execution client](./eth1.md) on the same machine - this will simulate the load of running a mainnet validator.
 
-To stress test it, `add--subscribe-all-subnets` to the [beacon node options](./options.md). This represents more or less the maximum load you could have on eth2.
+To stress test it, add `--subscribe-all-subnets` to the [beacon node options](./options.md). This simulates the maximum load that the consensus layer will put on the machine should you run 64 validators or more on it.
 
 ### How do I add an additional validator?
 
-To add an additional validator, just follow [the same steps](./keys.md) as you did when you added your first. You'll have to restart the beacon node for the changes to take effect.
+To add an additional validator, follow [the same steps](./keys.md) as you did when you added your first. You'll have to restart the beacon node for the changes to take effect.
```
Comment (zah) [inline docs/the_nimbus_book/src/faq.md:56]: A link to the Keymanager API can be inserted here.

File: docs/the_nimbus_book/src/docker.md (hunk 1 above)
Comment (arnetheduck) [inline docs/the_nimbus_book/src/docker.md]:
```suggestion
docker run -it --rm -v ${PWD}/data:/home/user/nimbus-eth2/build/data statusim/nimbus-eth2:amd64-latest --data-dir=build/data/shared_mainnet_0 --network=mainnet [other options]
```

Comment (arnetheduck) [reply docs/the_nimbus_book/src/docker.md]: good catch

File: docs/the_nimbus_book/src/faq.md (hunk 2 above)
Comment (arnetheduck) [reply docs/the_nimbus_book/src/faq.md:56]: I'm thinking that the raw keymanager API is a bit too messy for regular users, for which this page is targeted - would be better to do the CLI first

**Pull request merged**